docs/cli-usage.md

```diff
@@ -190,5 +190,42 @@ The best way to find out how that works is playing around with different values
 
 !!! note
 
-    Some identifiers do have text transformation enabled by default. 
+    Some identifiers do have text transformation enabled by default.
     Nevertheless make sure to **explicitly set** text transformation styles if you need them! All text transformation may be disabled by default in a future release of the generator.
+
+### Default Identifier Style settings
+(Please note that this list is work in progress)
+#### C++
+
+| Argument | Default  |
+| -------- | -------- |
+| `--ident-cpp-enum` | FooBar |
+| `--ident-cpp-field` | foo_bar |
+| `--ident-cpp-method` | foo_bar |
+| `--ident-cpp-type` | FooBar |
+| `--ident-cpp-enum-type` | FOO_BAR |
+| `--ident-cpp-type-param` | fooBar |
```

Comment:
```suggestion
| `--ident-cpp-enum-type` | FooBar |
| `--ident-cpp-type-param` | FooBar |
```
https://github.com/cross-language-cpp/djinni-generator/blob/0a54020e9b40efdb6e3e1ac04522866e109d14b2/src/main/scala/djinni/generator.scala#L120

Reply:
and indeed you seem to be right, Leandro.
```
class CppIdentStyle(ty: IdentConverter, enumType: IdentConverter, typeParam: IdentConverter,
                           method: IdentConverter, field: IdentConverter, local: IdentConverter,
                           enum: IdentConverter, const: IdentConverter)
```

(named parameter would really help readability in that case ...)

Reply:
Good idea! Here's a pull request for that :)
https://github.com/cross-language-cpp/djinni-generator/pull/32

Reply:
> Good idea! Here's a pull request for that :)
> #32

wow, now that is amazing! Thanks Leandro!!